cumulusci/robotframework/pageobjects/PageObjects.py

```diff
@@ -79,11 +80,16 @@ def __init__(self, *args):
         importer = robot.utils.Importer()
 
         for file_path in args:
-            try:
-                importer.import_class_or_module_by_path(os.path.abspath(file_path))
-                logger.debug("imported page object {}".format(file_path))
-            except Exception as e:
-                logger.warn(str(e))
+            path = self._find_file_in_pythonpath(file_path)
+            if path:
+                try:
+                    importer.import_class_or_module_by_path(os.path.abspath(path))
+                    logger.debug(f"imported page object {path}")
+                except Exception as e:
+                    logger.warn(str(e))
+            else:
+                logger.warn(f"Unable to find page object file '{file_path}'")
```

Comment:
```suggestion
                logger.warning(f"Unable to find page object file '{file_path}'")
```

And same question about warning vs exception